pkg/templates/externalccm/ccm.go

```diff
@@ -69,7 +69,7 @@ func waitForInitializedNodes(s *state.State) error {
 
 	s.Logger.Info("Waiting for nodes to initialize by CCM…")
 
-	return wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
+	return wait.Poll(5*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
Why did you increase the timeout this much?

Reply:
Sometimes cloud API is flaky (like in DO case, but could happen to anyone).